src/dsp.jl

```diff
@@ -410,3 +411,64 @@ function mfilter(r, s)
   n = length(r) - 1
   sfilt(f, padded(s, (0, n)))[n+1:end]
 end
+
+"""
+$(SIGNATURES)
+Inverse Short Time Fourier Transform (ISTFT). 
+
+The signal windowing must obey the constraint of "nonzero overlap add" (NOLA). 
+Implementation based on Hristo (2019, 2020) and `istft` in `scipy`. 
+
+H. Zhivomirov. On the Development of STFT-analysis and ISTFT-synthesis Routines 
```

Comment:
Unclear from the comment that this is a reference. Use consistent style as used elsewhere in the package?

Reply:
Currently I have included references only in the code, not the docs:
https://github.com/org-arl/SignalAnalysis.jl/blob/94b235a85b9c5be16a84c4782cea00a2d54c55aa/src/tfa.jl#L73
but I'm open to us changing that throughout and including in docs if you think we should do that. I just would like to be consistent throughout.

Reply:
Fixed.